app/src/routes/register/register-form.vue

```diff
@@ -81,18 +79,20 @@ async function onSubmit() {
 		/>
 		<v-input
 			v-model="password"
-			type="password"
+			:type="passwordHidden ? 'password' : 'text'"
 			autocomplete="new-password"
 			:placeholder="t('password')"
 			:disabled="isLoading"
-		/>
-		<v-input
-			v-model="passwordVerification"
-			type="password"
-			autocomplete="new-password"
-			:placeholder="t('confirm_password')"
-			:disabled="isLoading"
-		/>
+		>
+			<template #append>
+				<v-icon
+					v-tooltip="passwordHidden ? 'Show password' : 'Hide password'"
```

Comment:
All text should use i18n placeholders and be added to the `en-US.yaml` translation file (only that one, since Crowding takes care of the rest), similar to `"t('confirm_password')"`

Reply:
And then we can also remove `confirm_password` since it should be unused. Again, just removing it from `en-US.yaml` is enough (learned that myself, recently 🤓 )